internal/log/log.go

```diff
@@ -61,9 +61,9 @@ func New(logToStdout bool) Logger {
 	return logger
 }
 
-// Close closes the logger.
+// Close closes the logger. Ignore errors.
 func (l Logger) Close() {
-	l.ops.Close()
+	_ = l.ops.Close()
```

Comment:
internal/log: Return logger close error 
b8d1ddf019d3a52961d496a7b3e2fb23109f22d9

missing lint details in commit message.